coordinator/src/services/translation/common/index.ts

```diff
@@ -12,6 +13,10 @@ export function getMessageId(identifier: Array<fhir.Identifier>, fhirPath: strin
   )
 }
 
+export function identifyMessageType(bundle: fhir.Bundle): string {
+  return getMessageHeader(bundle).eventCoding?.code
+}
+
```

Comment:
moved to consume from pact project without introducing hapi request.logger dependency to have to mock out